README.md

```diff
@@ -1,2 +1,39 @@
 # saritasa-pre-commit-hooks
-Repository to store different Saritasa git pre-commit hooks 
+
+This repository contains special pre-commit hooks that can be reused for Github workflows.
+
+### Hooks available
+
+#### `check-nginx-wide-range`
+
+Prevent commiting of nginx wide range configs without `disabled` locations, i.e.
+
+```
+# below you can see the example of `wide` nginx config, because files would be searched
+# in `$uri` and `$uri/` directories, which can lead to opening of hidden files to the Internet
+location / {
+  try_files $uri $uri/ /index.php?$query_string;
```

Comment:
Hi Ksusha so my idea was only to test that in the case we find the following str in the config file(s)

```ini
try_files $uri $uri/ /index.php?$query_string;
```

if we find `$uri $uri/`  - we error out.

### I'm not sure that we need to check for existence of all `deny` rules below.

# Another concern

What if we have a project where let's say configuration

`location /app` is required.

so the deny rule

```
location ~ ^/(vendor|src|tests|vagrant|docs|phpunit|svn|git|docker|migrations|Makefile) {return 403;}
```

will be something like this.

What will happen then?

Reply:
Hi @dmitry-mightydevops 

1. We have lots of projects where we shouldn't fully forbid `try_files $uri $uri/` directives, otherwise frontend won't work correctly - nmbl-frontend, crossingguard-backend, ats-dart-frontend and etc, check out this search - https://github.com/search?q=org%3Asaritasa-nest+try_files+%24uri+%24uri%2F++path%3Anginx.conf&type=code So sometimes we need to have `try_files $uri $uri/` to make frontend work. That's why I added error raising only in case when required `disabled` locations are not set.
2. We can add `ignore_paths` or `disabled_locations_ignore` or smth like this, so user will pass values that should be ignored for disabled_locations. Currently webhook will raise error in the case you described, but we can add the adjustment I suggested.